applications/video_streaming/README.md

```diff
@@ -0,0 +1,473 @@
+unzip -o holoscan_client_cloud_streaming_v0.2/holoscan_client_cloud_streaming.zip -d holoscan_client_cloud_streaming
+
+# Clean up
+rm -rf holoscan_client_cloud_streaming_v0.2
+```
+
+### Server Dependencies
+
+Download the server streaming binaries from NGC:
+
+```bash
+# Navigate to the server operator directory  
+cd <your_holohub_path>/operators/video_streaming/streaming_server_enhanced
+
+# Download using NGC CLI
+ngc registry resource download-version "nvidia/holoscan_server_cloud_streaming:0.2"
+unzip -o holoscan_server_cloud_streaming_v0.2/holoscan_server_cloud_streaming.zip -d holoscan_server_cloud_streaming
+
+# Clean up
+rm -rf holoscan_server_cloud_streaming_v0.2
+```
+
+## Running the Applications
+
+The unified application provides both client and server applications.
+
+> [!IMPORTANT] Both client and server applications require Holoscan SDK 3.5.0. Set the SDK version environment variable before running the applications in each terminal, or use the `--base-img` option to specify the base image.
+>
+> ```bash
+> # Set SDK version environment variable
+> export HOLOHUB_BASE_SDK_VERSION=3.5.0
+> ```
+
+> [!NOTE] The client requires OpenSSL 3.4.0, which is installed inside the custom Dockerfile.
+
+### 1. Start the Streaming Server
+
+```bash
+./holohub run video_streaming
+```
+
+### 2. Start the Streaming Client (in another terminal)
+
+- **Option A: V4L2 Camera (Webcam)**, which uses `streaming_client_demo.yaml` and captures video from webcam with 640x480 resolution.
+
+  ```bash
+  ./holohub run video_streaming client_v4l2
+  ```
+
+- **Option B: Video Replayer**, which uses `streaming_client_demo_replayer.yaml` and replays a pre-recorded video file with 854x480 resolution.
+
+  ```bash
+  ./holohub run video_streaming client_replayer
+  ```
+
+## Python Applications
+
+Both server and client applications are available in Python using the Holoscan Python bindings. The Python implementations are fully compatible with their C++ counterparts and can be used interchangeably.
+
+### Requirements
+
+- Python 3.8 or higher
+- Holoscan SDK 3.5.0+ with Python bindings
+- Build with `-DHOLOHUB_BUILD_PYTHON=ON` flag
+- Custom Dockerfile with OpenSSL 3.4.0
+
+### Running Python Server
+
+```bash
+# From holohub root directory - runs with default settings (854x480 @ 30fps)
+./holohub run video_streaming server_python \
+  --docker-file applications/video_streaming/Dockerfile \
+  --docker-opts='-e EnableHybridMode=1' \
+  --configure-args='-DHOLOHUB_BUILD_PYTHON=ON'
+```
+
+**Default Configuration:**
+- Port: 48010
+- Resolution: 854x480
+- Frame Rate: 30 fps
+- Pipeline: StreamingServerUpstreamOp → StreamingServerDownstreamOp (passthrough/echo mode)
+
+### Running Python Client
+
+**Video Replayer Mode (Default - 854x480):**
+```bash
+# From holohub root directory - runs with video file playback
+./holohub run video_streaming client_python \
+  --docker-file applications/video_streaming/Dockerfile \
```

Comment:
This shouldn’t be necessary here, as the CLI should search for the app-specific Dockerfile first. However, if the CLI fails to find it, similar to the issue I’ve filed [here](https://github.com/nvidia-holoscan/holohub/issues/1177), please ensure that the `”dockerfile”` field is included in the `metadata.json`.